charts/consul/templates/partition-init-job.yaml

```diff
@@ -86,10 +86,10 @@ spec:
                 {{- if .Values.global.tls.enabled }}
                 -use-https \
                 {{- if not .Values.externalServers.useSystemRoots }}
-                -consul-ca-cert=/consul/tls/ca/tls.crt \
+                -ca-file=/consul/tls/ca/tls.crt \
```

Comment:
These two flags are already available in `c.http` so we will use those instead of having two similar, unnecessary flags.